tests/misc/cleanup_artifactory.py

```diff
@@ -0,0 +1,33 @@
+# File to delete all repositories on the server
+# that start with `unittest-`.
+#
+# NOTE:
+# * it is important to only call this script
+#   when no test pipeline is running
+# * you need to install `audfactory` to execute this script
+
+import audbackend
+import audfactory
+
+
+name = 'artifactory'
+host = 'https://audeering.jfrog.io/artifactory'
+
+r = audfactory.rest_api_get(f'{host}/api/repositories')
```

Comment:
To remove the dependency to `audfactory` we can do:

```suggestion
import requests

import audbackend


name = 'artifactory'
host = 'https://audeering.jfrog.io/artifactory'

username, api_key = audbackend.core.artifactory._authentication(host)
r = requests.get(f'{host}/api/repositories', auth=(username, api_key))
```

Reply:
Good idea, I changed it locally.